assets/queries/openAPI/unknown_prefix/query.rego

```diff
@@ -55,6 +56,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	schema_properties := doc.components.responses[r].content[c]
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)